contrib/akamai/controlbar/ControlBar.js

```diff
@@ -595,6 +603,22 @@ var ControlBar = function (dashjsMediaPlayer, displayUTCTimeCodes) {
             return content;
         },
 
+        getBrowserLocale = function() {
+          return (navigator.languages && navigator.languages.length) ? navigator.languages[0] : navigator.language;
+        },
+
+        getLabelForLocale = function (labels) {
+          const localeString = getBrowserLocale();
+
+          for (let i = 0; i < labels.length; i++) {
+            if (labels[i].lang && localeString.indexOf(labels[i].lang) > -1) {
```

Comment:
Maybe a nested loop of going through all the languages here? I see that Chrome and Firefox return `['en-US', 'en']`. Then you could just loop over `navigator.languages || [navigator.language]` or whatever.

I don't think it matters all that much, though, since the use of labels isn't widespread and the rest of the reference player isn't localized. I imagine that a more intelligent way of determining/setting the language for the reference player would be part of a broader effort for internationalization support.